lib/chai/core/assertions.js

```diff
@@ -607,24 +900,46 @@ module.exports = function (chai, _) {
   Assertion.addProperty('Arguments', checkArguments);
 
   /**
-   * ### .equal(value)
+   * ### .equal(val[, msg])
+   *
+   * Asserts that the target is strictly (`===`) equal to the given `val`.
+   *
+   *     expect(1).to.equal(1);
+   *     expect('foo').to.equal('foo');
+   * 
+   * Add `.deep` earlier in the chain to use deep equality instead. See the
```

Comment:
Many users get confused about the behavior of `deep` when it comes to object's prototypes, maybe it would be cool for us to explain more about it here and avoid issues later.